tests/BasicSearchTests.FunctionalTests.Core/TestSupport/V2SearchBuilder.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace BasicSearchTests.FunctionalTests.Core.TestSupport
+{
+    public class V2SearchBuilder
```

Comment:
Because these three builders are very similar, I would make a base class.

```
public class QueryBuilder
{
    protected readonly string Endpoint;

    public string Query { get; set; }
    public bool Prerelease { get; set; }

    protected virtual QueryString GetQueryString()
    {
        var queryString = System.Web.HttpUtility.ParseQueryString(string.Empty);
        queryString["q"] = Query;
        queryString["prerelease"] = Prerelease.ToString();
        return queryString;
    }
    
    public Uri RequestUri
    {
        get
        {
            return new Uri(Endpoint + GetQueryString(), UriKind.Relative);
        }
    }
}
```

Then set `Endpoint` in the subclass constructors and override `GetQueryString()` if necessary